packages/components/src/FormField/FormField.css

```diff
@@ -326,3 +337,22 @@
 .description {
   margin-top: var(--space-smaller);
 }
+
+.toolbar {
+  display: flex;
+  position: relative;
+  margin: var(--space-small) var(--field--padding-right) var(--space-small)
```

Comment:
Ah, this is the reason why that animation looks a bit choppy. it doesn't account for the margin. The `height: 0` on he animation doesn't remove the margin from that element.

Reply:
ah nice catch. removing that does make it feel smoother 100%

however then I need to add the margin in the animation, which is giving some interesting results of jumping around in a new way!
![Kapture 2024-04-09 at 11 13 44](https://github.com/GetJobber/atlantis/assets/11843215/39b48ccc-78d7-486a-a46c-3059b9b3fff6)

Reply:
i wonder if the margin just needs to not be on the animating element. would adding another wrapping div help?

Reply:
moved the margin to a separate div and let the animation be only responsible for height/opacity. feels much better now.